modules/base/rsnap.sh

```diff
@@ -3,11 +3,22 @@
 SNAP_SUFFIX="$3"
 SNAP_DIR="${CODE_DIRECTORY}/snapshots"
 CONTAINER=$4
+args=""
```

Comment:
I guess it should be `ARGS` since it is a "global" variable? :)

Reply:
You have seen nothing! :-)